src/System.Data.SqlClient/src/System/Data/SqlClient/SqlConnectionString.cs

```diff
@@ -521,7 +521,7 @@ internal string ObtainWorkstationId()
                 // permission to obtain Environment.MachineName is Asserted
                 // since permission to open the connection has been granted
                 // the information is shared with the server, but not directly with the user
-                result = string.Empty;
+                result = ADP.MachineName();
```

Comment:
Just curious: In .NET Framework, the length of the machine name is validated. Is the length check not needed for .NET Core?

https://github.com/Microsoft/referencesource/blob/master/System.Data/System/Data/SqlClient/SqlConnectionString.cs#L701